HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -80,6 +80,8 @@
       <sch:assert role='ERROR' test='count(h:Enclosure/h:Windows/h:Window) &gt;= 0'>Expected 0 or more element(s) for xpath: Enclosure/Windows/Window</sch:assert> <!-- see [Window] -->
       <sch:assert role='ERROR' test='count(h:Enclosure/h:Skylights/h:Skylight) &gt;= 0'>Expected 0 or more element(s) for xpath: Enclosure/Skylights/Skylight</sch:assert> <!-- see [Skylight] -->
       <sch:assert role='ERROR' test='count(h:Enclosure/h:Doors/h:Door) &gt;= 0'>Expected 0 or more element(s) for xpath: Enclosure/Doors/Door</sch:assert> <!-- see [Door] -->
+      <sch:assert role='ERROR' test='count(h:Enclosure/h:extension/h:PartitionWallMass) &gt;= 0'>Expected 0 or more element(s) for xpath: Enclosure/extension/PartitionWallMass</sch:assert> <!-- see [PartitionWallMass] -->
+      <sch:assert role='ERROR' test='count(h:Enclosure/h:extension/h:FurnitureMass) &gt;= 0'>Expected 0 or more element(s) for xpath: Enclosure/extension/FurnitureMass</sch:assert> <!-- see [FurnitureMass] -->
```

Comment:
@yzhou601 This should be `&lt;= 1` ("Expected 0 or 1 ...") so that multiple elements are not provided.